test/components/DayPickerKeyboardShortcuts_spec.jsx

```diff
@@ -135,6 +135,15 @@ describe('DayPickerKeyboardShortcuts', () => {
           expect(mockEvent.currentTarget.blur.callCount).to.equal(1);
         });
       });
+
+      describe('renderKeyboardShortcutsButton', () => {
+        it('renders the provided button', () => {
+          const props = { renderKeyboardShortcutsButton: () => (<button type="button">Success!</button>) };
+          const wrapper = shallow(<DayPickerKeyboardShortcuts {...props} />).dive();
+          const buttonWrapper = wrapper.children().find('button');
```

Comment:
possibly a more robust test would be to render a custom component, and `expect(wrapper.find(Custom)).to.have.lengthOf(1))`, rather than duplicating the text and element name here?

Reply:
The use case for renderKeyboardShortcutsButton is to literally send in a <button> though, should I still create a spec for a React Component?

Reply:
@kypan i mean, the purpose is to ensure that *whatever* `renderKeyboardShortcutsButton` returns is what's rendered; the only way to guarantee that is to use an unforgeable sentinel value - which would thus be a custom component.

Reply:
@ljharb That definitely makes sense. I've updated it. Thanks!